src/blocks/BulletPointBlock/BulletPointBlock.tsx

```diff
@@ -0,0 +1,217 @@
+    padding-left: 8px;
+    padding-right: 8px;
+  }
+
+  ${MOBILE_BP_UP} {
+    margin-left: -1.5rem;
+  }
+
+  ${LAPTOP_BP_UP} {
+    margin-left: -3.5rem;
+  }
+`
+
+const BulletPointTitle = styled(Heading)`
+  margin-bottom: 1rem;
+`
+
+const BulletPoint = styled.div<{
+  alignCenter: boolean
+  bulletPointLayout: boolean
+  columns: number
+}>`
+  display: flex;
+  flex-direction: ${(props) => (props.bulletPointLayout ? 'row' : 'column')};
+  width: 100%;
+  margin-top: 1.25rem;
+  margin-bottom: 1.25rem;
+
+  ${(props) =>
+    props.alignCenter &&
+    `
+      align-items: center;
+      text-align: center;
+    `}
+
+  ${MOBILE_BP_UP} {
+    flex-direction: column;
+    width: calc(50% - 1.5rem);
+    margin-left: 1.5rem;
+  }
+
+  ${TABLET_BP_UP} {
+    ${(props) =>
+      props.columns === 3 &&
+      `
+        width: calc((1 / 3 * 100%) - 1.5rem)
+      `}
+  }
+
+  ${LAPTOP_BP_UP} {
+    width: ${(props) => `calc((1 / ${props.columns} * 100%) - 3.5rem)`};
+    margin-left: 3.5rem;
+  }
+`
+
+const BulletPointHead = styled.div<{
+  alignCenter: boolean
+  bulletPointLayout: boolean
+}>`
+  display: flex;
+  align-items: center;
+  flex-shrink: 0;
+  margin-bottom: 1.5rem;
+
+  ${(props) =>
+    props.bulletPointLayout &&
+    `
+        justify-content: flex-start;
+        max-width: 2rem;
+        max-height: 2rem;
+        margin-right: 1.5rem;
+
+        ${MOBILE_BP_UP} {
+          max-width: none;
+          max-height: none;
+          width: auto;
+          margin-right: 0;
+        }
+      `}
+`
+
+const BulletPointImage = styled(DeferredImage)<{
+  alignCenter: boolean
+}>`
+  width: auto;
+`
+
+const BulletPointContent = styled.div``
+
+const BulletPointBody = styled.div<{
+  alignCenter: boolean
+  colorComponent: MinimalColorComponent
+}>`
+  max-width: ${(props) => (props.alignCenter ? '16rem' : 'none')};
+  margin-left: auto;
+  margin-right: auto;
+  color: ${(props) =>
+    getMinimalColorStyles(props.colorComponent?.color ?? 'standard').color};
+  font-size: 1rem;
+
+  & > * {
+    margin: 0 auto 1rem;
+  }
+
+  ${LAPTOP_BP_UP} {
+    font-size: 1.125rem;
+  }
+`
+
+export type BulletPointItemProps = ReadonlyArray<
+  BrandPivotBaseBlockProps & {
+    image: Image
+    title?: string
+    title_size: FontSizes
+    title_size_mobile?: FontSizes
+    paragraph: MarkdownHtmlComponent
+  }
+>
+
+export type BulletPointBlockProps = BrandPivotBaseBlockProps & {
+  align_center: boolean
+  bullet_point_layout: boolean
+  color_body: MinimalColorComponent
+  bullet_points: BulletPointItemProps
+}
+
+export const BulletPointBlock: React.FC<BulletPointBlockProps> = ({
+  align_center,
+  bullet_point_layout,
+  extra_styling,
+  color,
+  color_body,
+  size,
+  bullet_points,
+}) => (
+  <BulletPointSectionWrapper
+    brandPivot
+    colorComponent={color}
+    size={size}
+    extraStyling={extra_styling}
+  >
+    <ContentWrapper fullWidth>
+      <InnerWrapper>
+        {bullet_points.map((bullet) => (
```

Comment:
Maybe destructure? 😀 (Yes I'm obsessed with destructuring)
```suggestion
        {bullet_points.map(({ uuid, image, title, title_size, title_size_mobile, paragraph }) => (
```

Reply:
I agree that destructuring is the way to go most times, but when it's inside a component like I prefer to not do it. Because then I think it's easier to see where props comes from and it's a bit more condensed and easier to read :)

Reply:
Haha OK, we obviously have very different preferences for readability 🙃